Library/Homebrew/formula.rb

```diff
@@ -88,6 +88,9 @@ class Formula
 
   abstract!
 
+  # Used to track formulae that cannot be installed at the same time.
+  FormulaConflict = Struct.new(:name, :reason)
```

Comment:
This would be easier to understand as a `T::Struct`, but let's save that for a future refactor, as these are created via untyped paths: https://github.com/Homebrew/brew/blob/02947ea4edbdef5fcce9ee57fa289547f4d096c9/Library/Homebrew/formula.rb#L4097-L4101